common-rendering/src/components/FirstPublished.tsx

```diff
@@ -1,24 +1,26 @@
-import { css } from "@emotion/react";
-import { timeAgo } from "@guardian/libs";
-import { neutral, space, textSans } from "@guardian/source-foundations";
+import { css } from '@emotion/react';
+import { timeAgo } from '@guardian/libs';
+import { neutral, space, textSans } from '@guardian/source-foundations';
```

Comment:
This speaks to the remaining wider problem than anything in common-rendering is not being linted and is subject to the rules of whoever last edited it.

Reply:
The Guardian configs repo [recommends single quotes](https://github.com/guardian/configs/blob/main/packages/prettier/index.js#L9)

Reply:
Mostly concerned that it was wrong in the first place!

Reply:
Yeah, I see this quote war crop up pretty often and it seems like unnecessary diff noise. I guess the root issue is common not having a config